index.html

```diff
@@ -1,3 +1,6 @@
 <!DOCTYPE html>
     <h1>Hello World</h1>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="./index.js"></script>
+</body>
```

Comment:
Can you add the missing tags here? 
- opening html 
- opening and closing head tags with the script tags in there (ok to use `defer` for index.js)
- opening body